sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample10_RunningWithDifferentStorageVersion.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor.Samples.Infrastructure;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Storage.Blobs;
+
+/**
```

Comment:
This is interesting.   Not sure if there's a tooling consideration but the normal long-form comment in C# is `/*  ... */`.    The `/** ... */` notation is typically specific to JavaDoc comments.   Feels out of place here, though I don't have a strong opinion.